brayns/io/MatrixLoader.cpp

```diff
@@ -0,0 +1,276 @@
+/* Copyright (c) 2015-2017, EPFL/Blue Brain Project
+ * All rights reserved. Do not distribute without permission.
+ * Responsible Author: Jafet Villafranca Diaz <jafet.villafrancadiaz@epfl.ch>
```

Comment:
jafet? really?

Reply:
Yep, he initiated the work. I added me, but I am not removing him ;)